docs/config.md

```diff
@@ -178,6 +178,12 @@ endpoint or the `csv` one.
 
     LOG_NOT_FOUND = False
 
+#### MAX_QUERY_CHARS (int)
+In characters, max accepted length of the query. Prevent huge strings to be
```

Comment:
We need to document that default is 200 and that it's added to fight against DOS. So optional but highly recommended ;)